tests/test_learning.py

```diff
@@ -52,23 +52,33 @@ def test_decision_tree_learner():
 
 def test_neural_network_learner():
     iris = DataSet(name="iris")
-    iris.remove_examples("virginica")
-    
+
     classes = ["setosa","versicolor","virginica"]
-    iris.classes_to_numbers()
+    iris.classes_to_numbers(classes)
+
+    nNL = NeuralNetLearner(iris, [5], 0.15, 75)
+    pred1 = nNL([5,3,1,0.1])
+    pred2 = nNL([6,3,3,1.5])
+    pred3 = nNL([7.5,4,6,2])
 
-    nNL = NeuralNetLearner(iris)
-    # NeuralNetLearner might be wrong. Just check if prediction is in range.
-    assert nNL([5,3,1,0.1]) in range(len(classes))
+    # NeuralNetLearner might be wrong. If it is, check if prediction is in range.
+    assert pred1 == 0 or pred1 in range(len(classes))
```

Comment:
Thanks for the feedback, I appreciate it.

1) I'm not sure I fully understand this, is it possible to elaborate? I could write a function which, receiving a learner as input, will run the input learner and return the result, if that's what you are asking.

2) That is a great idea! I will get to work.